src/chain/blob.ts

```diff
@@ -0,0 +1,58 @@
+import {ABISerializableObject} from '../serializer/serializable'
+import {ABIEncoder} from '../serializer/encoder'
+import {arrayEquals, isInstanceOf} from '../utils'
+
+export type BlobType = Blob | string
+
+export class Blob implements ABISerializableObject {
+    static abiName = 'blob'
+
+    /**
+     * Create a new Bytes instance.
+     * @note Make sure to take a [[copy]] before mutating the bytes as the underlying source is not copied here.
+     */
+    static from(value: BlobType): Blob {
+        if (isInstanceOf(value, this)) {
+            return value
+        }
+        if (typeof value === 'string') {
+            return this.fromString(value)
+        }
+        throw new Error('Invalid blob')
+    }
+
+    static fromString(value: string) {
+        return new this(new Uint8Array(Buffer.from(value, 'base64')))
```

Comment:
Didn't catch this first time around but you cannot use Buffer in core, it's not available outside of node.js. You'll have to use atob and btoa (see https://developer.mozilla.org/en-US/docs/Glossary/Base64)

Reply:
Good catch. I originally swapped to a buffer because VSCode was complaining atob was deprecated 😅 

New snag... `atob` and `btoa` aren't available in nodejs 14, so our tests against that version are failing. I added back some checks to see if `Buffer` exists, and if so, use it.

https://github.com/wharfkit/antelope/pull/71/commits/a8526b71f75964b3f91f62c789badf4c6aef11c6